bot/ts/user.py

```diff
@@ -66,16 +66,15 @@ def client_id(self):
         if self._client_id is None:
             if self._unique_id is not None:
                 # easiest case: unique ID is set
-                self._client_id, ex = self.ts_conn.ts3exec(lambda t: t.query("clientgetids", cluid=self._unique_id).first().get("clid"), self._exception_handler)
+                self._client_id = self._ts_facade.client_ids_from_uid(client_uid=self._unique_id)[0].get("clid")  # use first and hope for the best.
             elif self._ts_db_id is not None:
-                self._unique_id, ex = self.ts_conn.ts3exec(lambda t: t.query("clientgetnamefromdbid", cldbid=self._ts_db_id).first().get("cluid"), self._exception_handler)
-                self._client_id, ex = self.ts_conn.ts3exec(lambda t: t.query("clientgetids", cluid=self._unique_id).first().get("clid"), self._exception_handler)
+                self._unique_id = self._ts_facade.client_get_name_from_dbid(client_dbid=self._ts_db_id).get("cluid")
+                self._client_id = self._ts_facade.client_ids_from_uid(client_uid=self._unique_id)[0].get("clid")  # use first and hope for the best.
```

Comment:
this doesn't seem to be in accordance with https://github.com/Xyaren/ts-gw2-verifyBot/blob/feature/ts-pooling/bot/ts/TS3Facade.py#L197 where the first element is already retrieved.

Reply:
The linked line retrieved the first part of the Tuple, while this line get's the first entry.

Reply:
I see.